src/components/PersonListItem/PersonListItem.tsx

```diff
@@ -0,0 +1,55 @@
+import { Card, CardActionArea, CardContent, Typography } from '@mui/material'
+import { ReactElement } from 'react'
+
+import { GetContacts } from '../../lib/queries/getContacts/getContacts'
+import { GetProcessDefinitions } from '../../lib/queries/getProcessDefinitions/getProcessDefinitions'
+
+interface PersonListItemProps {
+  contact: GetContacts
+  definitions: GetProcessDefinitions
+}
+
+export function PersonListItem({
+  contact,
+  definitions
+}: PersonListItemProps): ReactElement {
+  const processKeys = Object.keys(contact.process)
+  const processes = processKeys.map((key) => {
+    return definitions[key].title
+  })
+  const processLabel =
+    processes.length === 0
+      ? ''
+      : processes.length === 1
+      ? processes[0]
+      : processes.length === 2
+      ? `${processes.slice(0, -1).join(', ')} & ${processes.slice(-1)}`
+      : `${processes.slice(0, 2).join(', ')} & ${
+          processes.slice(2).length
+        } more`
```

Comment:
This was pretty hard to understand what was going on. Breaking into a switch statement would make it clearer. Also, there was no test for the case where processes are zero.

```suggestion
  let processLabel: string
  switch (processes.length) {
    case 0:
      processLabel = ''
      break
    case 1:
      processLabel = processes[0]
      break
    case 2:
      processLabel = `${processes.slice(0, -1).join(', ')} & ${processes.slice(
        -1
      )}`
      break
    default:
      processLabel = `${processes.slice(0, 2).join(', ')} & ${
        processes.slice(2).length
      } more`
  }
```